server/routes/admin/users.js

```diff
@@ -127,5 +127,25 @@ module.exports = ({ userAdminService }) => (router, audited) => {
     }
   }
 
+  router.get(
+    '/incomplete',
+    asyncMiddleware(async (req, res) => {
+      const incompleteUsers = await userAdminService.getIncompleteRoUsers(res.locals.token)
+      return res.render('admin/users/incompleteList', { incompleteUsers })
+    })
+  )
+
+  router.post('/incomplete/add', async (req, res) => {
+    try {
+      const input = req.body.incompleteUser
+      const userInput = JSON.parse(input)
+      req.flash('userInput', userInput)
+
+      return res.redirect('/admin/roUsers/add')
+    } catch (error) {
+      return res.redirect('/admin/roUsers/incomplete')
+    }
+  })
+
```

Comment:
This is redirecting to the existing "add user" page, passing the names and ID we already have so that you don't have to type it in again.